Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,130 @@
+# Glossary
+
+## Rook
+
+### CephBlockPool CRD
+
+The [CephBlockPool CRD](../CRDs/Block-Storage/ceph-block-pool-crd.md) is used by Rook to allow creation and customization of storage pools.
+
+### CephBlockPoolRadosNamespace CRD
+
+CephBlockPoolRados Namespace CRD is used by Rook to allow creation of Ceph BlockPool RadosNamespaces. For more information and examples refer to this [documentation](../CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md).
+
+### CephClient CRD
+
+CephClient CRD is used by Rook to allow [creation](https://rook.io/docs/rook/latest/CRDs/ceph-client-crd/) and updating clients.
+
+### CephCluster CRD
+
+CephCluster CRD is used by Rook to allow creation and customization of storage clusters through the custom resource definitions (CRDs). For more information about the different modes of creation of clusters, refer to our [documentation](../CRDs/Cluster/ceph-cluster-crd.md).
+
+### Ceph CSI
+
+Ceph CSI plugins implement an interface between a CSI-enabled Container Orchestrator (CO) and Ceph clusters. For further information, please check out [CSI Driver documentation](../Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md).
+
+### CephFilesystem CRD
+
+CephFilesystem CRD is used by Rook to allow creation and customization of shared filesystems through the custom resource definitions (CRDs). Check out the examples [here](../CRDs/Shared-Filesystem/ceph-filesystem-crd.md).
+
+### CephFilesystemMirror CRD
+
+CephFilesystemMirror CRD is used by Rook to allow [creation](../CRDs/Shared-Filesystem/ceph-fs-mirror-crd.md) and updating the Ceph fs-mirror daemon.
```

Comment:
On all the CRDs, let's have the link to their topics directly on the CRD name like this. Please also apply similar changes to other topics, even though I don't add suggestions on all of them.
```suggestion
The [CephFilesystemMirror CRD](../CRDs/Shared-Filesystem/ceph-fs-mirror-crd.md) is used by Rook to allow creation and updating the Ceph fs-mirror daemon.
```